RecoBTag/ONNXRuntime/plugins/ParticleNetSonicJetTagsProducer.cc

```diff
@@ -128,10 +129,39 @@ void ParticleNetSonicJetTagsProducer::acquire(edm::Event const &iEvent, edm::Eve
   edm::Handle<TagInfoCollection> tag_infos;
   iEvent.getByToken(src_, tag_infos);
   client_->setBatchSize(tag_infos->size());
+  skippedInference_ = false;
   if (!tag_infos->empty()) {
+
+    unsigned int maxParticles = 0;
+    unsigned int maxVertices = 0;
+    for (unsigned jet_n = 0; jet_n < tag_infos->size(); ++jet_n) {
+      if( ((*tag_infos)[jet_n]).features().get("pfcand_etarel").size() > maxParticles) maxParticles = ((*tag_infos)[jet_n]).features().get("pfcand_etarel").size();
```

Comment:
can be simplified a bit: `maxParticles = std::max(maxParticles, ...size())` etc.

Reply:
Done, though I had to cast the size() part as an unsigned int for the comparison